hazelcast/src/test/java/com/hazelcast/aggregation/CountAggregationTest.java

```diff
@@ -71,4 +71,40 @@ public void testCountAggregator_withAttributePath() {
 
         assertThat(result, is(equalTo(expectation)));
     }
+
+    @Test(timeout = TimeoutInMillis.MINUTE)
+    public void testCountAggregator_withNull() {
+        List<BigDecimal> values = sampleBigDecimals();
+        values.add(null);
+        long expectation = values.size();
+
+        Aggregator<Map.Entry<BigDecimal, BigDecimal>, Long> aggregation = Aggregators.count();
+        for (BigDecimal value : values) {
+            aggregation.accumulate(createEntryWithValue(value));
+        }
+
+        Aggregator<Map.Entry<BigDecimal, BigDecimal>, Long> resultAggregation = Aggregators.count();
+        resultAggregation.combine(aggregation);
+        long result = resultAggregation.aggregate();
+
+        assertThat(result, is(equalTo(expectation)));
+    }
+
+    @Test(timeout = TimeoutInMillis.MINUTE)
+    public void testCountAggregator_withAttributePath_withNull() {
+        List<Person> values = samplePersons();
+        values.add(null);
+        long expectation = values.size();
+
+        Aggregator<Map.Entry<Person, Person>, Long> aggregation = Aggregators.count("age");
+        for (Person person : values) {
+            aggregation.accumulate(createEntryWithValue(person));
+        }
+
+        Aggregator<Map.Entry<BigDecimal, BigDecimal>, Long> resultAggregation = Aggregators.count();Aggregators.count("age");
```

Comment:
`Aggregation.count();` is redundant here and shades the actual assignment to `Aggregation.count("age");`.
Same issue exists in existing code in `CountAggregationTest:68`